cmd/openshift-install/destroy.go

```diff
@@ -60,6 +60,11 @@ func runDestroyCmd(directory string) error {
 			return errors.Wrapf(err, "failed to destroy asset %q", asset.Name())
 		}
 	}
+	for _, asset := range installConfigTarget.assets {
```

Comment:
This seems to me like a hack to trick the installer into not using the state file. It assumes that the install config targets are ancestors of all of the other assets (which as an aside they are not). In my opinion, this is a fragile solution that does not afford much over deleting the state file entirely.

Reply:
Right. Check the alternate implementation in #1086